tests/v2/test_wakunode.nim

```diff
@@ -27,9 +27,13 @@ when defined(rln):
   import ../../waku/v2/protocol/waku_rln_relay/[waku_rln_relay_utils, waku_rln_relay_types]
 
 const RLNRELAY_PUBSUB_TOPIC = "waku/2/rlnrelay/proto"
+template sourceDir: string = currentSourcePath.parentDir()
+const KEY_PATH = sourceDir / "resources/key.pem"
```

Comment:
Sorry to be nitpicky but can we preface this with `test` like `test_key`? We really don't want people to accidentally end up using it in production. Ditto cert.

Reply:
changed in last commit